assets/styles/globalStyle.ts

```diff
@@ -67,6 +67,9 @@ const globalStyle = StyleSheet.create({
   marginTop30: {
     marginTop: verticalScale(30),
   },
+  marginRight20: {
```

Comment:
`name20` `name30` and etc. are bad names, what happends if the client wants to change the padding tomorrow from 20 to 25? You'd have to change the name of the variable as well, which forces a lot of changes in a lot of files and makes it difficult to review.